pkgs/development/python-modules/drms/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, pandas
+, six
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "drms";
+  version = "0.5.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "95cac0e14532893a44eeab8e329ddb76150e6848153d8cb1e4e08ba55569e6af";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    pandas
+    six
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  checkPhase = ''
+    python -m drms.tests
```

Comment:
${python.interpreter}

Reply:
Fixed. and stoping to add package haha. It's too easy with python-package-init